app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,7 @@
             <li class="dropdown">
               <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Home <span class="caret"></span></a>
               <ul class="dropdown-menu">
-                <li><a href="#">My Account</a></li>
+                <li><%# link_to "My Account", dashboard_path %></li>
                 <li><a href="#">Make A Donation</a></li>
                 <li role="separator" class="divider"></li>
                 <li><a href="#">Home</a></li>
```

Comment:
@pixapi @TheKaterTot @edilenedacruz What do you guys think about changing these links to fit with the rails link_to convention?